.github/actions/spelling/allow.txt

```diff
@@ -0,0 +1,382 @@
+abhaykatheria
+ableabhinav
+accountsservice
+adriennefriend
+Affero
+Aissue
+Akechi
+amd
+ananthan
+anthonyharrison
+Aopen
+api
+apk
+Args
+asn
+autoescape
+autoextract
+backend
+backport
+bcca
+bdbd
+berenjena
+Bienvenido
+bigbird
+binutils
+bksahu
+blog
+bluetooth
+bluetoothctl
+bluez
+brainwane
+brotlipy
+btn
+bubblewrap
+bugfixes
+busybox
+bzip
+cabextract
+CDNs
+cfea
+cff
+chaitanyamogal
+charset
+checkername
+chris
+cleartext
+clnt
+cmdline
+codecov
+combintation
+config
+copyleft
+cpio
+cpp
+CQA
+crond
+cryptsetup
+csrc
+css
+csv
+customisation
+customise
+customising
+cve
+cvedb
+cvedetails
+cvss
+cyberciti
+cygwin
+dbus
+debian
+debuginfo
+delimted
+devops
+dgst
+distro
+dnsmasq
+docstring
+DOCTYPE
+documentaiton
+dpkg
+dsa
+dtls
+enscript
+entrypoint
+Eqt
+Everyone
+everytime
+evp
+exe
+executables
+expat
+extrated
+fde
+fedoraproject
+ffmpeg
+figcaption
+filepaths
+filterdiv
+firefox
+freeradius
+FReeshabh
+frontend
+fsck
+fsf
+ftp
+fuction
+gcc
+getenv
+gettext
+Giridhar
+github
+glibc
+gnomeshell
+gnupg
+gnutls
+Google
+gpgme
+grep
+GSo
+gsoc
+gstreamer
+gupnp
+Habilidad
+Hacktoberfest
+haproxy
+Harmandeep
+haxx
+hdf
+hideme
+HMAC
+hostapd
+href
+Hrmn
+html
+http
+httpd
+hunspell
+hur
+icecast
+icu
+ikeydoherty
+img
+imgur
+imsahil
+informaci
+INI
+irssi
+itertools
+javascript
+jquery
+json
+Kamdar
+kbd
+kerberos
+kexectools
+kritirikhi
+landley
+lgpl
+libarchive
+libaries
+libbluetooth
+libbpg
+libc
+libcurl
+libdb
+libexpat
+libgcrypt
+libical
+libjpeg
+liblas
+libnss
+libpng
+libraryname
+libsndfile
+libsoup
+libsqlite
+libssh
+libtiff
+libvirt
+libvncserver
+libvorbis
+libvorbisenc
+libvorbisfile
+libxml
+libxslt
+lifecycle
+lighttpd
+linux
+localedir
+logrotate
+lua
+maintaiable
+malloc
+mananger
+Manjaro
+mariadb
+mariuszskon
+mcmilk
+mdadm
+mdc
+mdwood
+mearly
+meflin
+memcached
+metadata
+microsoft
+mkdir
+modulename
+mozilla
+msgid
+msgstr
+msi
+mtr
+myfork
+mysql
+Mystylesheet
+MYUSERNAME
+nano
+ncurses
+netpbm
+nginx
+ngnix
+Niraj
+nirajkamdar
+nitishsaini
+nlk
+noopener
+noreferrer
+nowdailynever
+nplurals
+ntp
+nvd
+OCSP
+onboarding
+oneline
+openafs
+openjpeg
+openldap
+openssh
+openssl
+opensuse
+openswan
+openvpn
+ossl
+pacman
+palletsprojects
+pcsc
+pdf
+pdxjohnny
+php
+pigz
+plotly
+png
+pocoo
+polarssl
+poppler
+postgresql
+Prajwal
+Prasath
+printf
+proivde
+prueba
+pspp
+Purvanshsingh
+pybabel
+pycqa
+pypa
+pypi
+pytest
+pythex
+pythonapp
+Qqe
+queryformat
+quickstart
+qyr
+radare
+Rahul
+readme
+readthedocs
+realpython
+regex
+rossburton
+rpmfile
+rst
+rsyslog
+sakshatshinde
+sanketsaurav
+sannanansari
+Saurabh
+sbs
+sdk
+shreyamalviya
+sobre
+somefile
+SPDX
+sqlite
+src
+srvr
+ssh
+ssl
+standalone
+statem
+stdout
+strngs
+strongswan
+sudo
+sunos
+suse
+svg
+svnv
+syslogd
+syslogng
+systemd
+SYSV
+tcpdump
+templating
+Tengo
+terri
+terriko
+testfiles
+timeline
+tls
+tmp
+TODO
+toml
+triaging
```

Comment:
Pretty sure this is a normal dictionary word whose usage predates tech (probably came from medicine), not sure why it had to be added.

Reply:
Yes, this surprised me too.